core/src/test/resources/HistoryServerExpectations/application_list_json_expectation.json

```diff
@@ -54,8 +54,8 @@
     "sparkUser" : "tgraves",
     "completed" : true,
     "appSparkVersion" : "3.0.0-SNAPSHOT",
-    "endTimeEpoch" : 1578764802615,
     "startTimeEpoch" : 1578764662851,
+    "endTimeEpoch" : 1578764802615,
```

Comment:
I don't think we have to do it here, but it'd be nice to sort the output keys. These diffs are almost all just lines moving around